app/__init__.py

```diff
@@ -38,5 +38,11 @@ def handleBaseException(error):
 api = Api(app)
 
 # All resources that need to be routed should be routed here.
+from app.users.resources import UserListResource, UserResource
+from app.authentication.resources import UserLogin, UserLogOut, UserInfo
+
 api.add_resource(UserListResource, '/users', '/users/')
 api.add_resource(UserResource, '/users/<int:userId>', '/users/<int:userId>/')
+api.add_resource(UserLogin, '/login/<int:userId>', '/login/<int:userId>/')
+api.add_resource(UserLogOut, '/logout', '/logout/')
+api.add_resource(UserInfo, '/me', '/me/')
```

Comment:
Feel free to disagree with me, but I think it would be great to group a resource with its set of urls. So for example:

```
# Users API
from app.users.resources import UserListResource, UserResource
api.add_resource(UserListResource, '/users', '/users/')
api.add_resource(UserResource, '/users/<int:userId>', '/users/<int:userId>/')

# Authentication API
from app.authentication.resources import UserLogin, UserLogOut, UserInfo
api.add_resource(UserLogin, '/login/<int:userId>', '/login/<int:userId>/')
api.add_resource(UserLogOut, '/logout', '/logout/')
api.add_resource(UserInfo, '/me', '/me/')
```